interfaces/builtin/network.go

```diff
@@ -56,6 +56,9 @@ dbus send
 
 @{PROC}/sys/net/core/somaxconn r,
 @{PROC}/sys/net/ipv4/tcp_fastopen r,
+
+# Allow using netcat as client
+/bin/nc.openbsd ixr,
```

Comment:
The rule as it stands is fine, however we should try to be cross-distro friendly. Please use this instead:
```
/{,usr/}bin/nc{,.openbsd} ixr,
```